tools/icalconverter/icalconverter.pro

```diff
@@ -1,7 +1,7 @@
 TEMPLATE=app
 TARGET=icalconverter
 QT-=gui
-CONFIG += link_pkgconfig
+CONFIG += link_pkgconfig timed-qt5
```

Comment:
Don't suppose this should be needed, timed only in the worker class which isn't used here?

Reply:
Mmh that's correct. I've removed it. I added by mistake, misunderstanding an error in another directory but printed just after the compilation of this tool…